common/src/main/java/com/wynntils/screens/statistics/widgets/StatisticButton.java

```diff
@@ -51,21 +52,20 @@ public void renderWidget(GuiGraphics guiGraphics, int mouseX, int mouseY, float
         CustomColor backgroundColor = getButtonBackgroundColor();
         RenderUtils.drawRect(poseStack, backgroundColor, this.getX(), this.getY(), 0, this.width, this.height);
 
-        int maxTextWidth = this.width - 2;
         FontRenderer.getInstance()
-                .renderText(
+                .renderScrollingText(
```

Comment:
I was thinking of implementing this to the activity screens and other places it could be used once it was merged into main so glad to see it here :)